x-pack/test_serverless/api_integration/test_suites/common/reporting/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Similar to how you do it for the functional tests, you have to add these tests to the `x-pack/test_serverless/api_integration/test_suites/*/common_configs`. Otherwise, these tests won't be picked up.